### ISSUE TITLE
Learning rate scaling duplicated in Dreambooth LoRa training script

### ISSUE THREAD

**Issue by stefan-canva** [huggingface/diffusers#3224]
### Describe the bug

The learning rate scaling is applied twice in the Dreambooth Lora training script, both in line 749 and in line 759. I assume this is not intentional:
https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/train_dreambooth_lora.py#L749

```
if args.scale_lr:
        args.learning_rate = (
            args.learning_rate * args.gradient_accumulation_steps * args.train_batch_size * accelerator.num_processes
        )

# Enable TF32 for faster training on Ampere GPUs,
# cf https://pytorch.org/docs/stable/notes/cuda.html#tensorfloat-32-tf32-on-ampere-devices
if args.allow_tf32:
    torch.backends.cuda.matmul.allow_tf32 = True

if args.scale_lr:
    args.learning_rate = (
        args.learning_rate * args.gradient_accumulation_steps * args.train_batch_size * accelerator.num_processes
    )
```

### Reproduction

No reproduction required, can be checked by viewing code.

### System Info

latest commit [20e426c](https://github.com/huggingface/diffusers/commit/20e426cb5dcd4baae5f16fffc24cb10a81264577)

**Comment by sayakpaul** [comment]
For sure, this is not intentional. Let me get https://github.com/huggingface/diffusers/pull/3180 merged and then we will fix it :) Thanks for reporting.
